src/openapi_python_generator/language_converters/python/common.py

```diff
@@ -24,6 +26,24 @@ def get_use_orjson() -> bool:
     return _use_orjson
 
 
+def set_dict_args(value: List[str]):
+    """
+    Set the value of the global variable _dict_args.
```

Comment:
Please try to find a bit more meaningful names for the parameter passed, and the documentation shouldn't just reflect the function name :) Instead, maybe document what these set_dict_args are, and where they are used.